src/Dropdown.js

```diff
@@ -274,10 +274,10 @@ class Dropdown extends React.Component {
       labelledBy: id,
       bsClass: prefix(props, 'menu'),
       onClose: createChainedFunction(
-        child.props.onClose, onClose, this.handleClose,
+        child.props.onClose, onClose, (event) => this.handleClose(event, 'close'),
```

Comment:
Why don't we like `close` here? It's too generic? I do think that its weird for the Dropdown to fire a `rootClose` type since it can't actually know that...

Bah yeah, the whole point is to differentiate rootClose from any other close, so yeah probably belongs in DropdownMenu. We should think about moving RootClose up out of the menu maybe in the longer term, enough people seem to expect it to live in Dropdown anyway.